imagepicker/src/main/java/com/esafirm/imagepicker/features/recyclers/RecyclerViewManager.java

```diff
@@ -166,10 +166,12 @@ public void setImageSelectedListener(OnImageSelectedListener listener) {
         imageAdapter.setImageSelectedListener(listener);
     }
 
-    public boolean selectImage() {
+    public boolean selectImage(boolean isSelected) {
         if (config.getMode() == ImagePicker.MODE_MULTIPLE) {
             if (imageAdapter.getSelectedImages().size() >= config.getLimit()) {
-                Toast.makeText(context, R.string.ef_msg_limit_images, Toast.LENGTH_SHORT).show();
+                if(!isSelected){
```

Comment:
Should be inlined with the limit checking

Reply:
OK, I'll fix it.